docs/source/en/using-diffusers/loading.md

```diff
@@ -60,6 +60,23 @@ repo_id = "runwayml/stable-diffusion-v1-5"
 pipe = StableDiffusionImg2ImgPipeline.from_pretrained(repo_id)
 ```
 
+You can use the application below to gauge the memory requirements of a pipeline you want to load beforehand without downloading the pipeline checkpoints:
```

Comment:
Tiny nit, but I think it'd be cool to call it a Space just so more people know what you can do with HF Spaces :)

```suggestion
You can use the Space below to gauge the memory requirements of a pipeline you want to load beforehand without downloading the pipeline checkpoints:
```